src/main/java/grocery/Grocery.java

```diff
@@ -10,19 +10,24 @@ public class Grocery {
     private String name;
     private int amount;
     private LocalDate expiration;
+    private String category;
     private double cost;
 
+
     /**
      * Constructs a Grocery.
      *
      * @param name Name.
      * @param amount Measurement of grocery.
      * @param expiration When grocery expires.
+     * @param category Category of grocery.
      */
-    public Grocery(String name, int amount, LocalDate expiration, double cost) {
+
+    public Grocery(String name, int amount, LocalDate expiration, String category, double cost) {
```

Comment:
The enhancements to the Grocery class to include category and cost are well thought out.